articles/connections/apple-siwa/configure-email-relay-service.md

```diff
@@ -0,0 +1,32 @@
+---
+title: Configure email relay service for Sign In with Apple
+description: Learn how to configure Apple's email relay service functionality for your app. 
+toc: true
+topics:
+  - authentication
+  - connections
+  - social
+  - apple
+  - email
+contentType: how-to
+useCase:
+  - add-login
+  - customize-connections
+  - configure-email-relay-service
+---
+## Configure Email Relay Service for Sign In with Apple
```

Comment:
This page is simply hosting instructions for Apple.  Are we in the business of doing that?  If so, some pictures might help.  LMK if you want me to snag some.

Reply:
I think we should delete this doc and just note that it needs to be done.